bufio.ts

```diff
@@ -162,6 +162,28 @@ export class BufReader implements Reader {
     return rr;
   }
 
+  /** reads exactly len(p) bytes into p.
+   * It returns the number of bytes copied and an error if fewer bytes were read.
+   * The error is EOF only if no bytes were read.
+   * If an EOF happens after reading some but not all the bytes,
+   * readFull returns ErrUnexpectedEOF. ("EOF" for current impl)
+   * On return, n == len(p) if and only if err == nil.
+   * If r returns an error having read at least len(buf) bytes,
+   * the error is dropped.
+   */
```

Comment:
Please include something like this in the docs: "Ported from https://golang.org/pkg/io/#ReadFull"